src/diffwofost/physical_models/crop/root_dynamics.py

```diff
@@ -134,13 +127,22 @@ class StateVariables(StatesTemplate):
         DWRT = Any(default_value=[torch.tensor(-99.0, dtype=DTYPE)])
         TWRT = Any(default_value=[torch.tensor(-99.0, dtype=DTYPE)])
 
-    def initialize(self, day, kiosk, parvalues):
+    def initialize(
+        self, day: datetime.date, kiosk: VariableKiosk, parvalues: ParameterProvider
+    ) -> None:
         """Initialize the model.
 
-        :param day: start date of the simulation
-        :param kiosk: variable kiosk of this PCSE  instance
-        :param parvalues: `ParameterProvider` object providing parameters as
-                key/value pairs
+        Args:
+            day (datetime.date): The starting date of the simulation.
+            kiosk (VariableKiosk): A container for registering and publishing
+                (internal and external) state variables. See PCSE documentation for
+                details.
+            parvalues (ParameterProvider): A dictionary-like container holding
+                all parameter sets (crop, soil, site) as key/value. The values are
+                arrays or scalars. See PCSE documentation for details.
+
+        Returns:
+            None: This method initializes the model and does not return anything.
```

Comment:
Maybe this can be removed (and similarly the returns for `calc_rates` and `integrate`)?

```suggestion
```